pkg/glusterutils/healinfo_gd1.go

```diff
@@ -0,0 +1,33 @@
+package glusterutils
```

Comment:
I would suggest heal info to be done in another PR and not mix in this.
Lets take the interface changes in one PR first and then send another PR for heal info.

Reply:
ack